### PR TITLE
Version 1.2.0

### PR DESCRIPTION
**Pull request by matq007**
Many thanks to contributing to nf-core/rnafusion!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If necessary, also make a PR on the [nf-core/rnafusion branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/rnafusion)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by matq007** [comment]
One more issue on fusion-report side: https://github.com/matq007/fusion-report/issues/30

**Comment by MaxUlysse** [comment]
I think we can get rid of both `download-singularity-img.nf` and `scripts/download-singularity-img.sh`, but apart from that, it's looks good to me.
Just need not to forget about bumping version at some point ;-)

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
-ENV PATH /opt/conda/envs/nf-core-rnafusion-1.1.0/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-rnafusion-dev/bin:$PATH
```
Comment (apeltzer) [inline Dockerfile:10]:
```suggestion
ENV PATH /opt/conda/envs/nf-core-rnafusion-1.2.0/bin:$PATH
```

File: Dockerfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ COPY environment.yml /
...
 
 # Dump the details of the installed packages to a file for posterity
-RUN conda env export --name nf-core-rnafusion-1.1.0 > nf-core-rnafusion-1.1.0.yml
+RUN conda env export --name nf-core-rnafusion-dev > nf-core-rnafusion-dev.yml
```
Comment (apeltzer) [inline Dockerfile:13]:
```suggestion
RUN conda env export --name nf-core-rnafusion-1.2.0 > nf-core-rnafusion-1.2.0.yml
```

File: containers/arriba/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+# Install the conda environment
+COPY environment.yml /
+RUN conda env create -f /environment.yml && conda clean -a
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/nf-core-rnafusion-arriba_1.2.0/bin:$PATH
+
+# Dump the details of the installed packages to a file for posterity
+RUN conda env export --name nf-core-rnafusion-arriba_1.2.0 > nf-core-rnafusion-arriba_1.2.0.yml
```
Comment (apeltzer) [inline containers/arriba/Dockerfile:14]: Here its fine :-D

**Comment by matq007** [comment]
Applying suggestions:

- [x] Remove singularity helper script
- [ ] Bump rnafusion to 1.2.0

**Comment by matq007** [comment]
Ok, it is still unclear for people to run the pipeline so I will update the README in a way that people fully understand it.

**Comment by matq007** [comment]
One last problem in the pipeline: The extra parameters are evaluated as `true` for some reason.

**Pull request merged**